zhaquirks/tuya/ts0601_gas.py

```diff
@@ -29,12 +32,9 @@
 class TuyaGasDetectorCluster(TuyaManufClusterAttributes):
     """Manufacturer Specific Cluster of the TS0601 gas detector."""
 
-    attributes = TuyaManufClusterAttributes.attributes.copy()
```

Comment:
Why to remove the `attributes.copy()`?
Regardless of the fact that it currently does nothing, in the future it may break if attributes are added to the parent classes.